plugins/router/f5/f5.go

```diff
@@ -663,6 +683,99 @@ func (f5 *f5LTM) ensureVserverHasIRule(vserverName, iRuleName string) error {
 	return nil
 }
 
+// checkPartitionPathExists checks if the partition path exists.
+func (f5 *f5LTM) checkPartitionPathExists(pathName string) (bool, error) {
+	glog.V(4).Infof("Checking if partition path %q exists...", pathName)
+
+	// Ensure pathname is absolute.
+	if !path.IsAbs(pathName) {
+		pathName = path.Join("/", pathName)
+	}
+
+	// F5 iControl REST API expects / characters in the path to be
+	// escaped as ~.
+	uri := fmt.Sprintf("https://%s/mgmt/tm/sys/folder/%s",
+		f5.host, strings.Replace(pathName, "/", "~", -1))
+
+	err := f5.get(uri, nil)
+	if err != nil {
+		if err.(F5Error).httpStatusCode != 404 {
+			glog.Errorf("partition path %q error: %v", pathName, err)
+			return false, err
+		}
+
+		//  404 is ok means that the path doesn't exist == !err.
+		return false, nil
+	}
+
+	glog.V(4).Infof("Partition path %q exists.", pathName)
```

Comment:
Kept this one and removed the one in ```ensurePartitionPathExists```. That way its easy to debug with the ```creating path and path exists``` messages.